src/Microsoft.AspNet.Buffering/HttpBufferingFeature.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNet.Http.Features;
+
+namespace Microsoft.AspNet.Buffering
+{
+    internal class HttpBufferingFeature : IHttpBufferingFeature
```

Comment:
Q : Curious what is the idea for exceptions thrown when serializers throw exceptions(example in MVC)? Like if we know the response is being buffered, then we can send back a friendly error message. But with this interface it doesn't seem to be possible to check 'if' the response is buffered...